File: 1832-CheckiftheSentenceIsPangram/1832-CheckiftheSentenceIsPangram.js
// Last updated: 3/17/2026, 10:54:22 AM
1/**
2 * @param {string} sentence
3 * @return {boolean}
4 */
5var checkIfPangram = function(sentence) {
6    let checkstr = "abcdefghijklmnopqrstuvwxyz";
7
8    for(let i=0;i<checkstr.length;i++){
9        if(!sentence.includes(checkstr[i])){
10            return false;
11        }
12    }
13    return true
14};

File: 1832-CheckiftheSentenceIsPangram/1832-CheckiftheSentenceIsPangram.cpp
// Last updated: 6/15/2026, 11:15:22 AM
1class Solution {
2public:
3    bool checkIfPangram(string sentence) {
4        string checkStr = "abcdefghijklmnopqrstuvwxyz";
5
6        for (int i = 0; i < checkStr.size(); i++) {
7            if (sentence.find(checkStr[i]) == string::npos) {
8                return false;
9            }
10        }
11
12        return true;
13    }
14};